### PR TITLE
fixing minimal PV size to be in Gi units and not G

### PR DESCRIPTION
**Pull request by romayalon**
Signed-off-by: Romy <romy2232@gmail.com>

### Explain the changes
1. Fixed the units of minimal PV size to be in Gi and not in G.

### Issues: Fixed #xxx / Gap #xxx
1. BZ #1873220

### Testing Instructions:
1.

### CODE REVIEW

**Review by guymguym — COMMENTED** [review]
@romayalon - Question - What about existing volumes which are smaller than the new minumum?
Any specific considerations are needed?

File: pkg/backingstore/reconciler.go [hunk 1]
```diff
@@ -504,7 +504,7 @@ func (r *Reconciler) ReadSystemInfo() error {
 			))
 		}
 		const defaultVolumeSize = int64(21474836480) // 20Gi=20*1024^3
-		const minimalVolumeSize = int64(16000000000) // 16G=16*1000^3
+		const minimalVolumeSize = int64(17179869184) // 16Gi=16*1024^3
```
[the remark this answers is not in the document]
Comment (romayalon) [reply pkg/backingstore/reconciler.go]: sure no problem, changes the defaultVolumeSize to be readable too.

**Comment by nimrod-becker** [comment]
@guymguym per Romy, will get into rejected.
But we don't have resize abilities today. 

Not sure its a real issue though

**Comment by guymguym** [comment]
@nimrod-becker @romayalon I just think we should be [sending out warnings](https://github.com/noobaa/noobaa-operator/blob/d06c993b1e18dc6a484b5da2295e7cade835185c/pkg/backingstore/reconciler.go#L189) if less than 10Gi but keep allowing as low as 10G, at least for a couple versions, because if someone manually picked the minimum value of 10G (which is not far fetched at all) then it will break and there's really no good reason it should.

**Comment by nimrod-becker** [comment]
@guymguym we were on 20GB two versions ago and 16G last version, I don't think we have 10 out there

**Comment by guymguym** [comment]
Ok but since we don't know how this was used in the wild, and don't want to break anyones deployment, we should strive for backwards compatibility, which means to just warn if below the new min 10Gi, and fail only if below the old min 10G.

**Comment by nimrod-becker** [comment]
@romayalon in the CLI , do we fail on the same threshold?

**Comment by nimrod-becker** [comment]
@romayalon what's the effort on pushing the entire check/validation into the create_hosts_pool func in pool_server and returning an appropriate error if < SIZE ?  (in all flows, CLI, reconcile)

**Review by guymguym — APPROVED** [review]
Summarizing the discussion me and @nimrod-becker had:

1. We are still compatible with pv-pools that worked before, we just fail sooner for sizes between `16G`-`16Gi` which was also eventually getting to an error (mode?) but we want to reject the spec earlier, before PV's are created and agent pods are started...

1. It seems like create_pool api in core should be checking this as well and return an error back if lower than limit. We seemed to have relied on the frontend checks for this limit before but the pool server is better off validating its own limitations.

So I think this PR is good, and we can followup in core.

**Pull request merged**